packages/admin/cms-admin/src/userPermissions/user/permissions/OverrideContentScopesDialog.tsx

```diff
@@ -112,12 +115,15 @@ export const OverrideContentScopesDialog: React.FC<FormProps> = ({ permissionId,
                                         type="checkbox"
                                         component={FinalFormCheckbox}
                                         value={JSON.stringify(contentScope)}
-                                        label={Object.entries(contentScope).map(([scope, value]) => (
-                                            <>
-                                                {camelCaseToHumanReadable(scope)}: {camelCaseToHumanReadable(value)}
-                                                <br />
-                                            </>
-                                        ))}
+                                        label={
+                                            label ||
+                                            Object.entries(contentScope).map(([scope, value]) => (
+                                                <>
+                                                    {camelCaseToHumanReadable(scope)}: {camelCaseToHumanReadable(value)}
+                                                    <br />
+                                                </>
+                                            ))
```

Comment:
Should we move this fallback label to the API? It seems to me that you wanted to do that with `UserPermissionsService#camelCaseToHumanReadable`, why did you decide against it?

One advantage would be that the content scope label handling would be completely in the API